README.md

```diff
@@ -3,5 +3,5 @@ An application to showcase and execute Splashkit games
 ## Install
 + Install the [SplashKit](https://splashkit.io) SDK using the [guide](https://splashkit.io/articles/installation/)
 ## Run
-+ Compile the application with the command ```skm clang++ program.cpp -o test -lstdc++fs```  
++ Compile the application with the command ```skm clang++ program.cpp -o test -Iinclude -lstdc++fs```  
```

Comment:
Just update this for good measure to avoid potential case-sensitivity issues. 
```program.cpp -> Program.cpp```

Reply:
Thanks @Delcari 😄 , I actually had no idea about `git mv`. Will go ahead and merge it then if no issues.